Tests/CSharp/ExpressionTests.cs

```diff
@@ -604,6 +605,129 @@ public void Test()
 }");
         }
 
+        [Fact]
+        public void MethodCallDictionaryAccessConditional()
+        {
+            TestConversionVisualBasicToCSharp(@"Public Class A
+    Public Sub Test()
+        Dim dict = New Dictionary(Of String, String) From {{""a"", ""AAA""}, {""b"", ""bbb""}}
+        Dim v = dict?.Item(""a"")
```

Comment:
I don't see a test case for plain `?("a")`, without the `Item` anywhere?

Shouldn't make a difference, but better to explicitly test it?